nixos/modules/services/web-apps/stirling-pdf.nix

```diff
@@ -0,0 +1,104 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.stirling-pdf;
+in
+{
+  options.services.stirling-pdf = {
+    enable = lib.mkEnableOption "the stirling-pdf service";
+
+    package = lib.mkPackageOption pkgs "stirling-pdf" { };
+
+    environment = lib.mkOption {
+      type = lib.types.attrsOf (lib.types.nullOr (lib.types.either lib.types.str lib.types.path));
```

Comment:
Why not allow `int`s, similar to e.g: https://github.com/NixOS/nixpkgs/pull/272918.

Reply:
Yeah that's a good idea.